packages/components/src/components/Divider/Divider.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Divider
+type: Component
+exportedName: Divider, DividerHeight
+---
+
+## API
```

Comment:
What about:
- Live example with dummy data (the "simple")
- Anatomy (show the different components in a simple way so we know what is available)
Example: 
```tsx
<Dialog>
  <DialogHeader></DialogHeader>
  <DialogContent></DialogContent>
  <DialogFooter></DialogFooter>
</Dialog>
```
- API
- Examples

Reply:
This is a Divider not a Dialog ^^

There aren't really any different components

Reply:
I know, I just propose having a generic and default structure

Reply:
agree with this then!